### DISCUSSION TITLE
issue in implementing qrcode using php 8.0.13

### DISCUSSION THREAD

**Discussion by georgefernandes** [chillerlan/php-qrcode#98] [Q&A]
Fatal error: Uncaught Error: Class "chillerlan\QRCode\Output\QROutputAbstract" not found in C:\xampp\htdocs\qrcode\vendor\chillerlan\php-qrcode\examples\MyCustomOutput.php:17 Stack trace: #0 {main} thrown in C:\xampp\htdocs\qrcode\vendor\chillerlan\php-qrcode\examples\MyCustomOutput.php on line 17

**Comment by codemasher** [comment]
Ok, so i verified whether the example works even on `dev-main`, so it's not a broken dependency, Also it looks that PHP is able to resolve the example namespace which is only available via composer's "require --dev" command. So another guess would be a missing autoloader include.
Also it would be less guessing if you could include a code example of yours that produces that output.
